### PR TITLE
Pl iss163

### PR DESCRIPTION
**Pull request by RedundantComputation**
Cleaning up a lot of redundant/unnecessary top bar elements so the interface looks a lot less cluttered. I also moved the functionality for loading a teleoscope into the Form Control, but it seems to be bugged when trying to index item 0 (maybe has something to do with default values). There is still currently a "Save teleoscope" button, but I'm planning to embed the save functionality into the "Orient towards" and "Orient away" buttons when I address the teleoscope data design issue, as per #173

### CODE REVIEW

File: backend/dispatch.py [hunk 1]
```diff
@@ -79,16 +79,6 @@ def handle_message(self, body, message):
                 },
             )
             res.apply_async()
-
-        if b['task'] == "save_UI_state":
```
Comment (pbucci) [inline backend/dispatch.py:83]: just wondering what this has been replaced with?

**Review by pbucci — CHANGES_REQUESTED** [review]
Looks overall good, but what's the idea with removing save_UI_state?

File: backend/dispatch.py (hunk 1 above)
Comment (RedundantComputation) [reply backend/dispatch.py:83]: It's being handled by the `save_teleoscope_state` function, although I plan on talking to Dhruv today about using class inheritance.

**Comment by RedundantComputation** [comment]
Superseded by #186 therefore closing.

**Pull request closed without merging**